package.json

```diff
@@ -6,9 +6,6 @@
     "packages": [
       "apps/*",
       "packages/*"
-    ],
-    "nohoist": [
-      "**/designer/**"
```

Comment:
I thought we made more workarounds haha

Reply:
The other things will be handle by the PR I made in the Vercel repo.